app/assets/stylesheets/application.scss

```diff
@@ -22,4 +22,44 @@ $hover-primary-color: #289cd1;
   &:hover { 
     background-color: $hover-primary-color !important; 
   }
-}
\ No newline at end of file
+}
+
+.pagination
+{
+  li {
+    a {      
+      background-color: white;
+      color: $primary-color !important;
+
+      &:focus {
+        color: #5a5a5a;
```

Comment:
store colors hash in a variable like primary color with descriptive names. And use that